.gitignore

```diff
@@ -6,7 +6,5 @@
 .idea/
 .vscode
 nbproject/
-Documentation-GENERATED-temp/
```

Comment:
Please keep this. It's used when rendering the documentation locally.

Reply:
I re-added the Documentation path.